src/libcore/ptr.rs

```diff
@@ -212,6 +212,14 @@ pub unsafe fn write_volatile<T>(dst: *mut T, src: T) {
 #[lang = "const_ptr"]
 impl<T: ?Sized> *const T {
     /// Returns true if the pointer is null.
+    ///
+    /// Example:
+    ///
```

Comment:
See http://doc.rust-lang.org/std/string/struct.String.html#method.new for an example.

(aaaand hopefully I can just point you at those guidelines I want to work on in the next few hours :smile: )